docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+
+## What is a product and what are Gatsby products?
+
+After a lot of back and forth, talking through examples, and working through questions, we landed on what we think of as “a product” at Gatsby.
+
+A product is an individual thing (software, hardware, etc.) that facilitates stable customer behaviors and is scalable.
+
+Additionally, a product must add value in order for a customer to continually use it. At Gatsby, we believe that we add value by solving real and impactful customer problems for individuals and teams in the web development community.
+
+## What is our product organization?
+
+A product organization is the group of people responsible for thinking about what we offer our customers and why. Our product organization, or product org for short, currently consists of a group of product managers, designers, and UX researchers.
+
+The goal of the product org is to understand the pain points of our customers, create relevant personas, and do the necessary research, discovery, and prototyping so that we can provide solutions they find valuable.
+
+## What does the product organization do?
+
+Since we are putting together a group focused on product, we also want to have outcomes that keep us on track. **An outcome measures the level of performance or achievement that occurred based on the thing we created and measured.** That means that we can have outcomes based around company goals, around products, or even around specific features.
+
+### Identifying the outcomes we want for persona and business problems
+
+We focus all of our conversations around problems at Gatsby. We want to discover, identify, and map out problems that exist both for the business and for our customers and community. By identifying how impactful and prevalent the problems are, we can start deciding if and when we want to tackle solving them.
+
+### Driving alignment with company Objectives and Key Results (OKRs)
+
+Identifying and writing down these problems also helps us write our objectives and key results because the objectives are created to list **what we want to do** based on the problem we are trying to solve, and the key results show **how we will measure that it’s been done**.
+
+Each team in Gatsby has objectives and key results that they are responsible for hitting.
+
+The product organization is responsible for making sure our key results are set and that any dependencies on other teams are clear, so they can successfully achieve their key results as well.
+
+### Creating an outcome-based roadmap that gives the company a sense of where we are with product development or features
+
+As we learn more about our community and our customers through research, discovery, and metrics, we will create a roadmap that shows what outcomes we want to provide for people. This is different from a traditional roadmap because we won’t be focusing on features, but rather on the problems we’ve seen and the outcome that would show we’ve solved the problem.
```

Comment:
maybe "will show we've solved the problem" since we're planning on reaching the outcomes and if we haven't at some point, we'll just keep trying until they're solved?

Reply:
Good catch! I've made the update!